test/Microsoft.Dnx.CommonTestUtils/DirTree.cs

```diff
@@ -153,6 +168,12 @@ public bool MatchDirectoryOnDisk(string dirPath, bool compareFileContents = true
             return true;
         }
 
+        private static string RemoveWhitespace(string actual)
+        {
+            actual = new string(actual.ToCharArray().Where(ch => !char.IsWhiteSpace(ch)).ToArray());
```

Comment:
How effective it is. Will the performance sucks when the file is really really huge?